.github/workflows/container.yaml

```diff
@@ -22,8 +22,17 @@ jobs:
       matrix:
         variant:
           - fakemachine-arch
-          - fakemachine-bullseye
-          - fakemachine-bookworm
+          - fakemachine-debian-bullseye
+          - fakemachine-debian-bookworm
```

Comment:
don't chagn the variant names

Reply:
right, i thought calling them `debian-*` would be clearer, but perhaps not :-)